### ISSUE TITLE
duplicate properties within a given selector

### ISSUE THREAD

**Issue by absfarah** [ju1ius/css-parser#5]
I noticed a small issue where if there are two css property declarations like so:

```
        #right_nav ul li a.current{
            padding:0px;
            padding:0 0 0 35px; 
         }
```

What will happen is that the returned expanded padding css property will all be 0 i.e:

```
   padding-left: 0; 
   padding-right: 0; 
   padding-top: 0; 
   padding-bottom 0;
```

So I investigated the `_canAddShorthandExpansion` function and it looks the condition on line: https://github.com/ju1ius/css-parser/blob/master/lib/ju1ius/Css/StyleDeclaration/ExpandShorthands.php#L271

Should be a `<` (less than). The smaller the number the further up the deceleration and the more important it is right?

Basically, what is happening is that the latter is not set as the final property. Another example would be:

```
margin: 10px 12 px 14px 16px;
margin: 30px;
```

The latter should be the final margin property. From my investigation, I could only narrow it down to that function and the greater than sign.
